src/Knp/Menu/Twig/Helper.php

```diff
@@ -88,7 +88,7 @@ public function get($menu, array $path = [], array $options = [])
      */
     public function render($menu, array $options = [], $renderer = null)
     {
-        $menu = $this->castMenu($menu);
+        $menu = $this->castMenu($menu, $options);
```

Comment:
this is now mixing the renderer options and the builder options, which has already been rejected in the past.

Reply:
that's ok 
but [here](https://symfony.com/doc/current/bundles/KnpMenuBundle/menu_builder_service.html) i have this opportunity.
I would like to build different menu depending on group. How to build it? create a lot different Menu Builders as Services

Reply:
thanks i've found solution knp_menu_get